docs/best-practices/api-design.md

```diff
@@ -328,7 +328,7 @@ A subsequent request from the client application can retrieve the remainder of t
 One of the primary motivations behind REST is that it should be possible to navigate the entire set of resources without requiring prior knowledge of the URI scheme. Each HTTP GET request should return the information necessary to find the resources related directly to the requested object through hyperlinks included in the response, and it should also be provided with information that describes the operations available on each of these resources. This principle is known as HATEOAS, or Hypertext as the Engine of Application State. The system is effectively a finite state machine, and the response to each request contains the information necessary to move from one state to another; no other information should be necessary.
 
 > [!NOTE]
-> Currently there are no standards or specifications that define how to model the HATEOAS principle. The examples shown in this section illustrate one possible solution.
+> Currently there are no general-purpose standards that define how to model the HATEOAS principle. While [The Atom Publishing Protocol](https://tools.ietf.org/html/rfc5023) is an established standard for providing hypermedia controls, it is (as the name suggests) primarily designed for content publishing and thus best fit for [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)-style APIs. While more general-purpose alternatives such as [HAL](http://stateless.co/hal_specification.html), [Siren](https://github.com/kevinswiber/siren), [Verbose](https://verbose.readthedocs.io/en/latest/index.html), [Mason](https://github.com/JornWildt/Mason), [Hyper](http://hyperjson.io/) and [Hydra](https://www.hydra-cg.com/) exist, none are ratified standards yet. The examples shown in this section illustrate one possible, proprietary solution.
```

Comment:
Any chance we can simplify to just?  We don't probably need to link up every hypermedia implementation out there.

```suggestion
> Currently there are no general-purpose standards that define how to model the HATEOAS principle.  The examples shown in this section illustrate one possible, proprietary solution.
```